spec/acceptance/listen_spec.rb

```diff
@@ -7,6 +7,15 @@
   let(:all_options) { base_options.merge(polling_options).merge(options) }
 
   let(:wrapper) { setup_listener(all_options, :track_changes) }
+  around do |example|
+    warn_level = $VERBOSE
+    begin
+      $VERBOSE = nil
+      example.run
+    ensure
+      $VERBOSE = warn_level
+    end
+  end
```

Comment:
The warnings are from the `Kernel.warn` that I added specifically for older Mac versions... so they would only show up in that case.

I was following the precedent from the BSD adapter in using `Kernel.warn` for that message rather than `Listen::Logger`.

On thinking it over I think it would be better to just take this out and the few contributors who might notice these warnings can just turn verbosity down from the command line if they really want to (`RUBYOPT='-W0' bundle exec rspec`). Thanks for the link on metadata though, good to know.